### PR TITLE
[cassandra] optionally wrap connections in ddtrace

### PR DESCRIPTION
**Pull request by talwai**
Add an option to `CassandraCqlManager` to trace queries with [ddtrace](https://github.com/Datadog/dd-trace-py)

### CODE REVIEW

File: beaker_extensions/cassandra_cql.py [hunk 1]
```diff
@@ -108,14 +111,15 @@ def __init__(self, namespace, url=None, keyspace=None, column_family=None,
 
         self._tries = int(params.pop('tries', 1))
 
-        cluster = self.__connect_to_cluster(url, params)
+        # Use a ddtrace-d connection if instructed to
```
Comment (dbenamydd) [inline beaker_extensions/cassandra_cql.py]: Stupid nit: could you remove this line?

**Comment by dbenamydd** [comment]
There's no way to use the new functionality unless there's a weird custom wrapper around the backend class like we have. Ie a normal user can't flip a setting in the .ini file to use it. I guess that's ok though. If anyone else actually want to use it, we can add that later.

🚢

File: setup.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
       zip_safe=False,
       install_requires=[],
       extras_require={
-          'cassandra_cql': ['cassandra-driver>=3.1.0'],  # 3.1.0 added result iterators
+          'cassandra_cql': ['cassandra-driver>=3.1.0','ddtrace>=0.3.4'],  # 3.1.0 added result iterators
```
Comment (dbenamydd) [inline setup.py]: Oh, want to limit this to `<1.0.0`? (I assume we're using semver for ddtrace.)

Comment (talwai) [inline setup.py]: :shrug: we will do semver, but i think we'll come out ahead of any breaking changes

**Pull request merged**